arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1650,7 +1655,15 @@
         },
         "action": {
           "description": "Action of the geo filter.",
-          "type": "string"
+          "enum": [
+		    "Block",
+            "Allow"
+		  ],
+		  "type": "string",
+		  "x-ms-enum": {
+			"name": "action",
```

Comment:
>action [](start = 12, length = 6)

Actions is very generic term please change name to more discoverable. e.g. GeoFilterActions